GameOfLifeApp/View/Display.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Displays 2 boolean values arrays. 
+    /// </summary>
+    public class Display : IDisplay
+    {
+        private IGameField pr;
+
+        public Display(IGameField properties)
```

Comment:
why for interface IGameField the object is properties?

Reply:
Resolved.